### ISSUE TITLE
Не работает скачивание PDF.

### ISSUE THREAD

**Issue by Ulyanov-programmer** [kurnakovv/Resume#27]
При попытке нажать на кнопку...

![download-file-resume herokuapp com_api_File](https://github.com/kurnakovv/Resume/assets/66691708/3aa5ec30-3e35-461a-8a48-751590691ce4)

**Comment by kurnakovv** [comment]
Да, уже год эта кнопка не работает и всё лень починить, как нибудь займусь :)
